app/src/main/res/layout/book_card.xml

```diff
@@ -24,7 +24,7 @@
                 android:layout_weight="0.5"
                 android:paddingRight="8sp"
                 android:text="BOOK NAME HERE"
-                android:textSize="16sp" />
+                android:textSize="24sp" />
```

Comment:
Tbh, i don't like how big the book card is now. I think you could just bold the book title and italicize the author to make it pop out more without making the cards massive. Eventually, we can colour code the book status messages so that we can still keep the cards small but easy to read.

Reply:
I just made the change to the xml